### PR TITLE
Init development/2.8

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
**Component**:

'kubernetes', 'build'

**Context**: 

Initialization of the branch development/2.8

**Summary**:

- Bump MetalK8s version
- Bump Kubernetes version to 1.19.6
- Bump coredns version to 1.7.0
- Bump etcd version to 3.4.13-0
- Deactivate downgrade test for minor version in the CI (See: #1750)

---

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -753,7 +753,9 @@ stages:
           name: Trigger upgrade and downgrade test stages simultaneously
           stage_names:
             - single-node-upgrade-centos
-            - single-node-downgrade-centos
+            # NOTE: Deactivate minor downgrade on this branch
+            # See: https://github.com/scality/metalk8s/issues/1750
+            #- single-node-downgrade-centos
```
Comment (gdemonet) [inline eve/main.yml:758]: We upgrade from 3.4.3 to 3.4.13, shouldn't downgrade work for patch versions? (I tried looking around their docs, but there doesn't seem to be mentions of downgrade)

**Review by gdemonet — APPROVED** [review]

File: eve/main.yml (hunk 1 above)
Comment (TeddyAndrieux) [reply eve/main.yml:758]: Hmm right, worth a try, I will merge this PR as is for the moment and I will check this :+1:

Comment (TeddyAndrieux) [reply eve/main.yml:758]: Btw I also forgot to disable the downgrade in salt in the defaults.yaml so anyway I will do another PR for downgrade (either to really disable it or to enable it back :smile: )

Comment (TeddyAndrieux) [reply eve/main.yml:758]: Patch version etcd downgrade is supported (at least from 3.4.13 to 3.4.3) so this commit get reverted in #3023

**Pull request merged**